etl/steps/data/garden/artificial_intelligence/2023-06-14/ai_robots.meta.yml

```diff
@@ -163,15 +163,22 @@ tables:
         description: "Industrial robots are defined as “automatically controlled, reprogrammable, multipurpose manipulator, programmable in three or more axes, which can be either fixed in place or mobile for use in industrial automation applications."
 
       number_of_industrial_robots_installed_2021:
-        title: Number of industrial robots installed by country
+        title: Total number of industrial robots installed in the world
         unit: 'robots'
         description: "Industrial robots are defined as “automatically controlled, reprogrammable, multipurpose manipulator, programmable in three or more axes, which can be either fixed in place or mobile for use in industrial automation applications."
         display:
           name: Number of industrial robots installed
```

Comment:
It's annoying, but it would be nice to add `numDecimalPlaces: 0` to the display section for all the indicators where the value will be an integer.